tpv/core/helpers.py

```diff
@@ -114,3 +114,13 @@ def tool_version_gte(tool, version):
 
 def tool_version_gt(tool, version):
     return parse_version(tool.version) > parse_version(version)
+
+
+def object_store_ids_and_dataset_size(job):
```

Comment:
Since this is returning a generic set of attributes of interest, maybe we can rename it as such?
```suggestion
def get_dataset_attributes(datasets):
```

Reply:
Sure! I am fine with it.